Config/BuildSettings.swift

```diff
@@ -249,6 +249,11 @@ final class BuildSettings: NSObject {
     static let roomCreationScreenRoomIsEncrypted: Bool = true
     static let roomCreationScreenAllowRoomTypeConfiguration: Bool = true
     static let roomCreationScreenRoomIsPublic: Bool = false
+    
+    // MARK: - Room Screen
+    
+    static let roomScreenAllowVoIPForDirectRoom: Bool = true
+    static let roomScreenAllowVoIPForConferenceRoom: Bool = true
```

Comment:
`roomScreenAllowVoIPForNonDirectRoom`. I was tempted by `roomScreenAllowVoIP` but it may be confusing.